File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.py
forms = {'c': 26, 'd': 10}
plate = input()

answer = forms[plate[0]]

for i in range(1, len(plate)):
    added = forms[plate[i]]
    if plate[i] == plate[i - 1]:
        added -= 1
    answer *= added

print(answer)

File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? 'dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const forms = { c: 26, d: 10 };
const plate = input[0];

let answer = forms[plate[0]];

for (let i = 1; i < plate.length; i++) {
  let added = forms[plate[i]];
  if (plate[i] === plate[i - 1]) {
    added -= 1;
  }
  answer *= added;
}

console.log(answer);